src/components/NotificationCarousel/NotificationCarousel.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ComponentBaseProps, generateRandomString } from '../../shared';
+import { Notification } from '../Notification/Notification';
+import { Wrapper, WrapperProps } from '../Wrapper';
+import { notificationSpacing } from './styles';
+
+/**
+ * Notification carousel component properties
+ * Extends html label attributes
+ * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes
+ */
+export interface NotificationCarouselProps
+  extends WrapperProps,
+    ComponentBaseProps<HTMLLabelElement>,
+    React.LabelHTMLAttributes<HTMLLabelElement> {
+  /**
+   * List of items (notifications) to stack to each other
+   */
+  carouselItems?: CarouselItem[];
+
+  /**
+   * Maximum height of the whole carousel component, so the notifications won't cover the whole view if there are many (only most recent ones would be visible)
+   */
+  maxHeight?: string;
```

Comment:
Mä ehkä tekisin niin, että voidaan parametrisoida kuinka monta ilmoitusta voi näkyä kerralla.

Reply:
Jättäisin ehkä maxHeight:in silti vaihtoehdoksi, jos on tilanteita missä ilmoituksien korkeus voi olla joskus todella suuri ja välillä pieni. Koitan tehdä lisäksi lukumäärällä rajaamisen (uusi vaihtoehto.)

Reply:
Max heightiä pystyy säätämään esim style-attribuutin kautta, niin sitä ei tarvita erikseen. Jos sisällön määrän rajaa yhteen tai kahteen ilmoitukseen niin sillä pystyy hoitamaan ahtaan tilan, kuten mobiilin.